### PR TITLE
feat(applications): add event broadcasts nsip-project case team (BOAS-1434)

### PR DESCRIPTION
**Pull request by BenSurgisonPINS**
## Describe your changes

- Added a relation between Case and ProjectTeam in the prisma schema.
- Added an option to retrieve the related project team when getting the case data from the db.
- Amended building the nsip-project event data to include the project team data when available.
- Updated the api unit tests to support the changes.
- Added the ability to simulate the AD user data with dummy user data to make it easier to develop locally where AD is unavailable.

I have also tested the functionality above manually.
Reseeding has been tested to make sure it is unaffected.

## BOAS-1434 Add Event Broadcasts - nsip-project case team
https://pins-ds.atlassian.net/browse/BOAS-1434

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/.env.example [hunk 1]
```diff
@@ -132,3 +132,19 @@ AZURE_BLOB_STORE_ACCOUNT_KEY=
...
 
+
+## Development #################################################################
+
+
+# Optional location of a json file containing dummy AD user data when AUTH_DISABLED=true
+# An example of one user in the file is as follows:
+# [
+#   {
+#     "id": "1",
```
Comment (DavidJWhiteDev) [inline apps/web/.env.example:143]: can this be clear that it will be a guid, eg '1111-2222-3333'

**Review by DavidJWhiteDev — COMMENTED** [review]
also check that re-seeding the DB works, as I know it clears down the case records etc in dev and Test, and we have had issues of the seeder failing after a schema change.

Otherwise all looks good, and I like the idea of the dummy data to enable us to easily work in local dev, nice one!

File: apps/web/.env.example (hunk 1 above)
Comment (BenSurgisonPINS) [reply apps/web/.env.example]: I've changed the id to a guid in the example as suggested.

**Comment by BenSurgisonPINS** [comment]
> also check that re-seeding the DB works, as I know it clears down the case records etc in dev and Test, and we have had issues of the seeder failing after a schema change.
> 
> Otherwise all looks good, and I like the idea of the dummy data to enable us to easily work in local dev, nice one!

Reseeding tested and works fine.

**Review by DavidJWhiteDev — APPROVED** [review]

**Pull request merged**